addons/mail/static/src/core/common/message.xml

```diff
@@ -5,6 +5,7 @@
         <div t-if="message.isNotification" class="o-mail-NotificationMessage text-break mx-auto text-muted opacity-75 px-3 text-center smaller" t-on-click="onClickNotificationMessage"  t-att-class="props.className" t-ref="root">
             <i t-if="message.notificationIcon" t-attf-class="{{ message.notificationIcon }} me-1"/>
             <span class="o-mail-NotificationMessage-author d-inline" t-if="authorName and !message.body.includes(escape(authorName))" t-esc="authorName"/> <t t-out="message.body"/>
+            <span class="o-mail-Message-date o-xsmaller ms-1" t-out="message.dateSimpleWithDay"/>
```

Comment:
The `.opacity-75` should apply on the content of message notification but not the date, otherwise the datetime have lower opacity than messages which is not good:
<img width="1404" alt="Screenshot 2025-03-13 at 18 32 17" src="https://github.com/user-attachments/assets/7eef4ab5-bb9f-4216-b51f-6633be8b1dfe" />